common/k8stest/zfs.go

```diff
@@ -0,0 +1,80 @@
+package k8stest
+
+import (
+	"encoding/json"
+	"fmt"
+	"os/exec"
+	"strings"
+
+	"github.com/openebs/openebs-e2e/common"
+	"github.com/openebs/openebs-e2e/common/controlplane"
+)
+
+// GetZFSVolume returns a ZFSVolume CR by name
+func GetZFSVolume(name string) (*common.ZFSVolume, error) {
+	return controlplane.GetZFSVolume(name)
+}
+
+// ListZFSVolumes returns all ZFSVolume CRs in the mayastor namespace
+func ListZFSVolumes() ([]common.ZFSVolume, error) {
+	return controlplane.ListZFSVolumes()
+}
+
+// GetZFSVolumeStatus returns the status.state of a ZFSVolume by name
+func GetZFSVolumeStatus(name string) (string, error) {
+	return controlplane.GetZFSVolumeStatus(name)
+}
+
+// GetZfsVolume gets a ZFS volume using direct kubectl command
+func GetZfsVolume(name, namespace string) (*common.ZFSVolume, error) {
```

Comment:
Rename this and below function with something like GetZfsVolumeCr

Reply:
updated the function name